backend/src/v4/models/intercom.js

```diff
@@ -46,14 +46,15 @@ Intercom.setIntercomHash = (userProfile) => {
 		.digest("hex");
 };
 
-Intercom.createContact = async (external_id, name, email, subscribed, company /* , job_title, phone_number, industry, found_us*/) => {
+Intercom.createContact = async (external_id, name, email, subscribed, company, createdAt /* , job_title, phone_number, industry, found_us*/) => {
 	if (!accessToken) {
 		return;
 	}
 
 	const custom_attributes = {subscribed /* , job_title, industry, found_us*/ };
 	if (company) {
 		custom_attributes.company_entered = company;
+		custom_attributes.company = company;
```

Comment:
is there much point in doing this? Shouldn't we just have one field?

Reply:
`company_entered` removed.